go.mod

```diff
@@ -568,7 +568,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-require github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
+require (
+	github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
+	k8s.io/kube-aggregator v0.23.5
```

Comment:
could you move this new line part of the `require` ligne 38  https://github.com/DataDog/datadog-agent/pull/16570/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L38
because we other `k8s.io` deps in it.

Reply:
Addressed by 167afdfbbdbdbfb5e75966395d0f70231de52214